src/lib/seen-this-session.js

```diff
@@ -0,0 +1,26 @@
+import auth from 'panoptes-client/lib/auth';
+
+const subjectsSeenThisSession = [];
+
+auth.listen('change', () => {
+  return subjectsSeenThisSession.splice(0);
+});
+
+function addAlreadySeen(workflowId, subjectIds) {
```

Comment:
Wait, hang on, is this ever called?

Reply:
Oh, it should've been in the classifications duck. I might've forgotten to save that file before pushing the commit.